.github/workflows/deploy.yml

```diff
@@ -76,6 +76,26 @@ jobs:
           message: |
             Review app deployed to ${{ steps.deploy.outputs.url }}
 
-  # deploy staging
+  deploy_staging:
+    name: Deploy staging
+    needs: [ docker ]
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/deploy-to-staging'
```

Comment:
Needs to be changed to `refs/heads/main` once we check that the branch is working.